packages/web/src/components/premium-content-purchase-modal/components/PurchaseContentFormFooter.tsx

```diff
@@ -91,7 +91,8 @@ export const PurchaseContentFormFooter = ({
     [title]
   )
 
-  const { amountDue } = purchaseSummaryValues
+  const { totalPrice } = purchaseSummaryValues
+  console.debug('REED amountDue', totalPrice)
```

Comment:
console.debug (protip: leave these as console.logs if they're not meant to be in production so that lint will catch them!)